### PR TITLE
[docker] Remove traces of config-builder

### PR DESCRIPTION
**Pull request by davidiw**
* Remove validator-dynamic, the pinacle of config-builder
* Remove building / copying config-builder
* Remove shell scripts as they almost all use config-builder and we execute commands directly within the container
* Stripped out most of the mint test code but we'll add that back via a proper e2e test in compose in the near future... that test wasn't really ... testing anything

### CODE REVIEW

**Review by rexhoffman — APPROVED** [review]
Nicely done.   A few files may conflict with what I landed, but just in comments.

File: docker/mint/Dockerfile [hunk 1]
```diff
@@ -39,53 +39,18 @@ RUN mkdir -p /opt/libra/bin  /libra/client/data/wallet/
...
-    COUNTER=$(( $COUNTER - 1 )); \
-    done; \
-    if [ -z $PIDS ]; then exit 1; else kill -9 $PIDS || true; fi
-
 FROM pre-test as prod
 # Mint proxy listening address
 EXPOSE 8000
 
-# Define CFG_SEED, AC_HOST, AC_PORT and CFG_CHAIN_ID environment variables when running
-CMD /opt/libra/bin/docker-run.sh
```
Comment (sausagee) [inline docker/mint/Dockerfile:87]: We can remove the config-builder but not this startup script yet. I think the faucet server is using the CLI based implementation.  cc @mgorven

Comment (mgorven) [reply docker/mint/Dockerfile:87]: k8s doesn't use any of these wrapper scripts, this is fine to remove.

**Review by mgorven — APPROVED** [review]

**Review by sausagee — APPROVED** [review]

File: docker/mint/Dockerfile (hunk 1 above)
Comment (davidiw) [reply docker/mint/Dockerfile:87]: no one is using these :/
they probably have some value in indicating what is going on, but i would propose we just update the Dockerfile to have an explicit command at some point if that's the route we want to support.... alternatively we just leverage docker-compose and slowly get that into CI. That way we ensure that our ops infra has better test coverage.

Comment (sausagee) [reply docker/mint/Dockerfile:87]: docker compose sounds like a good follow-up.

**Comment by sausagee** [comment]
@davidiw the LBT issue is fixed, you can retry land.

**Comment by davidiw** [comment]
> @davidiw the LBT issue is fixed, you can retry land.

There's a couple other hairy issues that I'm trying to weed out at the same time :P, hopefully this latest tweak gets the rest of them... and then we can finally say good bye to the scary side of config-builder.

**Comment by bmwill** [comment]
Sooooo Bors only listens to one command (the first one) per message. This is something that could easily be changed but we just didn't get around to it. I manually added the hi-pri label though

**Comment by davidiw** [comment]
Does bors rebase this on current master for lbt and other tests or does it use this as is? Because I fear I didn't properly rebase this to pass lbt :/

**Comment by bmwill** [comment]
@davidiw Bors always rebases onto the latest master prior to running tests

**Comment by davidiw** [comment]
sweet

**Pull request closed without merging**